js/components/lab-model.js

```diff
@@ -20,8 +20,10 @@ export default class LabModel extends PureComponent {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.props.model !== prevProps.model) {
-      this.setState({loading: true});
+    if (this.props.model !== prevProps.model && !this.props.liveData) {
+      this.setState({ loading: true });
+    } else {
+      this.setState({ loading: false });
```

Comment:
This is really strange. Sensor changes temperature exactly in the same way as the slides does - just calling `setState({temperature: newValue});`. And when I move slider, nothing unexpected happens. Reading previous lines, it seems to me that loading is set to true only if model is updated. It might be something else going on or I miss something here.